developer-docs-site/docs/whats-new-in-docs.md

```diff
@@ -5,6 +5,12 @@ slug: "whats-new-in-docs"
 
 # Whats New in Docs
 
+This page shows the key updates to the developer documentation on this site.
+
+## 13 October 2022
+
+- Added [user documentation](/docs/guides/use-aptos-explorer.md) for [Aptos Explorer](https://explorer.aptoslabs.com/) to Aptos.dev covering common use cases and popular Explorer screen descriptions.
```

Comment:
"to Aptos.dev" feels unnecessary. Nit. Approving.

Reply:
Just trying to make it clear these can't be found on the Explorer itself.